roles/ansible-service-broker-setup/defaults/main.yml

```diff
@@ -12,7 +12,11 @@ broker_mobile_configuration:
   value: true
 
 #The url for the registry that hosts our APB images
+ansible_playbookbundle_registry_type: dockerhub
 ansible_playbookbundle_registry_url: https://registry.hub.docker.com
-
 #The organisation that our APB images are in
 ansible_playbook_bundle_registry_org: aerogearcatalog
+rhcc_registry_auth_name: ""
+rhcc_registry_auth_type: ""
+rhcc_registry_tag: "{{ rhcc_release_tag }}"
+rhcc_registry_url: "https://registry.access.redhat.com"
```

Comment:
I'm not sure if this should be registry.access.redhat.com or registry.redhat.io. Probably the latter, but I'll see if I can check.